### ISSUE TITLE
Study object(s) on add a new study seems to have no function

### ISSUE THREAD

**Issue by Gerd-ECRIN** [ecrin-github/rms-portal-new#91]
The field is empty without a possibility to add something.

**Comment by cudillal** [comment]
The list of study object(s) is composed of all the data objects that have this study as "parent study". Perhaps we could add at least a placeholder text if there are no DOs, and even the possibility of associating DOs from there (with a warning saying this will change the parent study of the DO).

**Comment by Gerd-ECRIN** [comment]
This would change the whole workflow, if DOs are added within Study Page. Perhaps it can be simple hidden for adding new studies?

**Comment by cudillal** [comment]
You're right, I didn't notice it was showing when adding a study, it should definitely by hidden.

**Comment by Gerd-ECRIN** [comment]
Is hidden now
